src/lib/blueprint.ts

```diff
@@ -1,3 +1,5 @@
+import { openapi } from '@seamapi/types/connect'
```

Comment:
I know this is just a test, but instead of importing this in this file and added this to this file

```
const blueprint = createBlueprint({ openapi })
console.log(JSON.stringify(blueprint, null, 2))
```

The existing snapshot test should already do exaclty this, and even better, will show up in the PR diff. Try removing the extra code from this file and running `npm test:update` instead :)